oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -235,10 +235,9 @@ storage:
     superDatasetShardsFactor: ${SW_STORAGE_BANYANDB_SUPERDATASET_SHARDS_FACTOR:2}
     concurrentWriteThreads: ${SW_STORAGE_BANYANDB_CONCURRENT_WRITE_THREADS:15}
     profileTaskQueryMaxSize: ${SW_STORAGE_BANYANDB_PROFILE_TASK_QUERY_MAX_SIZE:200} # the max number of fetch task in a request
-    streamBlockInterval: ${SW_STORAGE_BANYANDB_STREAM_BLOCK_INTERVAL:4} # Unit is hour
-    streamSegmentInterval: ${SW_STORAGE_BANYANDB_STREAM_SEGMENT_INTERVAL:24} # Unit is hour
-    measureBlockInterval: ${SW_STORAGE_BANYANDB_MEASURE_BLOCK_INTERVAL:4} # Unit is hour
-    measureSegmentInterval: ${SW_STORAGE_BANYANDB_MEASURE_SEGMENT_INTERVAL:24} # Unit is hour
+    blockIntervalHours: ${SW_STORAGE_BANYANDB_BLOCK_INTERVAL_HOURS:24} # Unit is hour
+    segmentIntervalDays: ${SW_STORAGE_BANYANDB_SEGMENT_INTERVAL_DAYS:1} # Unit is day
+    overrideGroupIntervals: ${SW_STORAGE_BANYANDB_OVERRIDE_GROUP_INTERVALS:""} # For example, group1,4,1;group2,2,2
```

Comment:
What is `overrideGroupIntervals`?

Reply:
> What is `overrideGroupIntervals`?

Mentioned in issue https://github.com/apache/skywalking/issues/10102, we have to Support setting the interval rule based on the group name more than global defaults.

Reply:
Do you mean a thing similar with index specific settings in elasticsearch?

```
    # Specify the settings for each index individually.
    # If configured, this setting has the highest priority and overrides the generic settings.
    specificIndexSettings: ${SW_STORAGE_ES_SPECIFIC_INDEX_SETTINGS:""}
```

Reply:
> Do you mean a thing similar with index specific settings in elasticsearch?
> 
> ```
>     # Specify the settings for each index individually.
>     # If configured, this setting has the highest priority and overrides the generic settings.
>     specificIndexSettings: ${SW_STORAGE_ES_SPECIFIC_INDEX_SETTINGS:""}
> ```

I think so

Reply:
Then we should rename it to `specificGroupSettings`. And the value format should be in JSON for better extension in the future. What do you think?

Reply:
> Then we should rename it to `specificGroupSettings`. And the value format should be in JSON for better extension in the future. What do you think?

Make sense to me